### ISSUE TITLE
Needs better documentation

### ISSUE THREAD

**Issue by oriansj** [oriansj/mes-m2#3]

**Comment by MagicRB** [comment]
It does, my question is, whether this works or not. Because I can't get to work, even with cloning nyacc to the root of the repo.

**Comment by oriansj** [comment]
Yes it does work, however the big problem is that it is extremely touchy about nyacc versions.
You need to use: https://download.savannah.gnu.org/releases/nyacc/nyacc-1.00.2.tar.gz

the full and exact steps needed for the building of mes and having it self-host are right here: https://github.com/fosslinux/live-bootstrap/blob/master/sysa/mes/mes.kaem

and here are the steps for taking that self-hosted binary to build TCC: https://github.com/fosslinux/live-bootstrap/blob/master/sysa/tcc-0.9.26/tcc-0.9.26.kaem

**Comment by MagicRB** [comment]
thanks! I'll get back to packaging this with Nix, I'll then try to create a doc of sorts, but it will be a bit blog post-y probably I'm much more comfortable with that style of writing.

**Comment by oriansj** [comment]
that is perfectly fine. Thank you for your help.
